internal/commands/pause_notify.go

```diff
@@ -87,13 +87,16 @@ func PauseNotifyIncidentByDialog(
 		pauseNotifyTime sql.NullTime
 	)
 
+	logWriter := app.Logger.With(
+		log.NewValue("channelID", channelID),
+		log.NewValue("pauseNotifyTimeText", pauseNotifyTimeText),
```

Comment:
pauseNotifyReasonText value is also already available at this point, so we put it in the logWriter fields